appIconIndicators.js

```diff
@@ -680,6 +680,36 @@ class UnityIndicator extends IndicatorBase {
     constructor(source) {
         super(source);
 
+        if (!UnityIndicator.defaultProgressBarTrack) {
+            // singleton with default values
+            UnityIndicator.defaultProgressBarTrack = {
```

Comment:
IIRC you can do the same at class level doing something like:

`static get defaultProgressBarTrack() { return .... }` so that's initialized just once anyways.

Reply:
Even simpler, with an static property.